### ISSUE TITLE
Need to pass --settings parameter to be able to access public settings on the client

### ISSUE THREAD

**Issue by darkbasic** [Urigo/meteor-client-bundler#23]
I tried to set an evironment variable just in case:
```
{
  "runtime": {
    "DDP_DEFAULT_CONNECTION_URL": "http://domain.tld:3000",
    "ROOT_URL": "http://domain.tld:80",
    "METEOR_SETTINGS": "{\"public\": {\"facebook\": {\"permissions\": [\"basic_info\", \"user_interests\", \"user_activities\", \"read_friendlists\"], \"profileFields\": [\"name\", \"gender\", \"location\"]}}}"
  },
  "import": [

  ]
}
```
but obviously it didn't work because meteor expects a --settings parameter and doesn't work with an environment variable.

**Comment by darkbasic** [comment]
By the way I don't think that having to specify a settings parameter a second time in the client is a good design decision. Meteor should probably publish public settings using DDP instead. I don't know the innings so excuse me if I don't know how it's supposed to work.

**Comment by lnmunhoz** [comment]
@darkbasic Did you figure it out how to pass settings to the bundler?

**Comment by darkbasic** [comment]
No, unfortunately I didn't looked at a possible solution, I just worked it around.

**Comment by bo01ean** [comment]
https://github.com/Urigo/Ionic2CLI-Meteor-WhatsApp/issues/73#issuecomment-352637869

Can I add this if anyone is interested?

**Comment by bazzila** [comment]
try this

{
    "runtime": {
    "DDP_DEFAULT_CONNECTION_URL": "http://domain.tld:3000",
        "ROOT_URL": "http://domain.tld:80",
        "PUBLIC_SETTINGS": {
        "facebook": {
            "permissions": ["basic_info", "user_interests", "user_activities", "read_friendlists"],
            "profileFields": ["name", "gender", "location"]
        }
    }
},
    "import": []
}
